### PR TITLE
Widget checkbox fixed edit button and added LastUpdate component to content

### PR DESCRIPTION
**Pull request by marcin-witaszek**

### CODE REVIEW

File: cogboard-webapp/src/components/LastUpdate.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
 import React from 'react';
-import {Typography} from "@material-ui/core";
+import styled from '@emotion/styled/macro';
```
Comment (goeson00) [inline cogboard-webapp/src/components/LastUpdate.js]: Empty line between libs imports and components imports.

File: cogboard-webapp/src/components/LastUpdate.js [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+import { Typography } from "@material-ui/core";
+
+const StyledTypography = styled(Typography)`
```
Comment (goeson00) [inline cogboard-webapp/src/components/LastUpdate.js]: Move styled to own file.

File: cogboard-webapp/src/components/LastUpdate.js (hunk 1 above)
Comment (marcin-witaszek) [reply cogboard-webapp/src/components/LastUpdate.js]: Done

File: cogboard-webapp/src/components/LastUpdate.js (hunk 2 above)
Comment (marcin-witaszek) [reply cogboard-webapp/src/components/LastUpdate.js]: Done

**Comment by marcin-witaszek** [comment]
I also removed legacy required propType for this widget that was redundant and causing additional JS error in the console

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget/index.js [hunk 3]
```diff
@@ -21,14 +20,10 @@ const CheckboxWidget = props => {
   };
 
   return (
-    <WidgetIconButton aria-label="Toggle status" onClick={handleChangeStatus}>
-      <StatusIcon size="large" status={status}/>
+    <WidgetIconButton aria-label="Toggle status" data-cy={`checkbox-${status}`} onClick={handleChangeStatus}>
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/components/widgets/types/CheckboxWidget/index.js]: do we really want `data-cy` to change with the status?

Comment (marcin-witaszek) [reply cogboard-webapp/src/components/widgets/types/CheckboxWidget/index.js]: Good point, I made it constant and added `aria-checked` attribute to be in sync with status

**Pull request merged**